SingularityService/src/main/java/com/hubspot/singularity/data/RequestManager.java

```diff
@@ -632,11 +639,20 @@ public void activateLeaderCache() {
     if (useWebCache && webCache.useCachedRequests()) {
       return webCache.getRequests();
     }
+
+    if (requestsCache.isEnabled()) {
+      List<SingularityRequestWithState> requests = requestsCache.get("all");
+      if (requests != null) {
+        return requests;
+      }
+    }
```

Comment:
I wonder if we should just not use the web cache here if this is the case. Seems overkill to have the leaderCache -> web cache -> zk cache all here. Lots of duplicate memory usage to store it all 3 times

Reply:
As discussed offline, to have ease in revert, the removal of web cache will be in a separate PR